src/features/onboarding/accountPassphraseWarning/AccountPassphraseWarningScreen.tsx

```diff
@@ -43,7 +44,12 @@ const AccountPassphraseWarningScreen = () => {
         variant="primary"
         width="100%"
         marginBottom="l"
-        onPress={() => navigation.push('AccountPassphraseGeneration')}
+        onPress={() =>
+          navigation.push('AccountPassphraseGeneration', {
+            netType: NetTypes.MAINNET,
```

Comment:
Does `MAINNET` need to be hardcoded here?

Reply:
Yeah, we aren't allowing testnet as of now but I have a branch that configures it for testing. This makes it future proof so that you can pass the NetType, but right now its just hard coded to mainnet